scripts/esbuild/mock-doc.ts

```diff
@@ -0,0 +1,73 @@
+import type { BuildOptions as ESBuildOptions } from 'esbuild';
+import fs from 'fs-extra';
+import { join } from 'path';
+
+import { writeSizzleBundle } from '../bundles/plugins/sizzle-plugin';
+import { bundleDts } from '../utils/bundle-dts';
+import { BuildOptions } from '../utils/options';
+import { writePkgJson } from '../utils/write-pkg-json';
+import { getBaseEsbuildOptions, getEsbuildAliases, runBuilds } from './util';
+
+export async function buildMockDoc(opts: BuildOptions) {
+  const inputDir = join(opts.buildDir, 'mock-doc');
+  const srcDir = join(opts.srcDir, 'mock-doc');
+  const outputDir = opts.output.mockDocDir;
+
+  // bundle d.ts
+  await bundleMockDocDts(opts, inputDir, outputDir);
+
+  writePkgJson(opts, outputDir, {
+    name: '@stencil/core/mock-doc',
+    description: 'Mock window, document and DOM outside of a browser environment.',
+    main: 'index.cjs',
+    module: 'index.js',
+    types: 'index.d.ts',
+    sideEffects: false,
+  });
+
+  const mockDocAliases = getEsbuildAliases();
+
+  const sizzlePath = await writeSizzleBundle(opts);
+  mockDocAliases['sizzle'] = sizzlePath;
+
```

Comment:
Should the `parse5` bundle be aliased here as well? It's in [our current configuration](https://github.com/ionic-team/stencil/blob/9d2d0a0612caafb88c559907c6dec1a39e760615/scripts/bundles/mock-doc.ts#L52), and I'm not sure if/why we should omit it here

Reply:
ah yeah that's just an oversight, we do want to include it

Reply:
added that in